spec/features/view_ip_address_readiness_spec.rb

```diff
@@ -0,0 +1,46 @@
+# TODO: don't require eveerything ever to fill in a form
+require 'features/support/not_signed_in'
+require 'features/support/sign_up_helpers'
+require 'features/support/errors_in_form'
+require 'features/support/activation_notice'
+require 'support/notifications_service'
+require 'support/confirmation_use_case'
+require 'timecop'
+
+describe 'even more requires' do
+  include_examples 'confirmation use case spy'
+  include_examples 'notifications service'
+
+  after { Timecop.return }
```

Comment:
I have always found not returning `Timecop` to be one of the biggest causes of random spec failures in a test suite. I wonder how much slower the specs would be if we threw this in `rails_helper.rb`?

Reply:
Inclined to do this when we have `Timecop` in multiple places. Then I'd try it out to see the speed change.